README.md

```diff
@@ -34,6 +41,14 @@ so as to avoid system-specific differences in linting behaviour.
 Note: If you're working on an Angular(JS) application, consider using
 [eslint-plugin-angular](https://github.com/Gillespie59/eslint-plugin-angular).
 
+### Auto-fixing Linting Issues
+
+Some ESLint rules support automated fixing. To apply fixes, run `eslint` with
+the [`--fix`](https://eslint.org/docs/user-guide/command-line-interface#--fix)
+flag.
```

Comment:
Is `--fix` always a good idea to use? especially in combination with prettier.

Maybe we should just recommend prettier and leave `eslint --fix` out of the docs?

Reply:
They don't address the same issues so using both may be useful and when configured properly (the prettier ESLint configs should disable formatting-related linting rules) they should not (a careful "should not") interfere.

Reply:
However, `eslint --fix` would likely be run once, – manually – to get the project into shape and not automatically (e.g. on each commit) like prettier.

Reply:
Maybe it makes sense to clarify this? If this makes sense to you that is…